setup.py

```diff
@@ -16,6 +16,10 @@
     "deprecation>=2.0.0",
     # Progress bars
     "tqdm>=4.48.1",
+    # Platform-independent file locking
+    "filelock~=3.0",
```

Comment:
I wish we pinned to `>=3.0` like all our other dependencies (both here and in signac).

Reply:
Pros: We wouldn't have to update until something breaks
Cons: CI could break unexpectedly when a major version bump changes APIs. You could view this as a pro though since it means we'd be alerted sooner rather than getting stuck on an older version forever.

I'd be fine changing.